.github/workflows/subflow_run_e2e.yaml

```diff
@@ -0,0 +1,64 @@
+on:
+  workflow_call:
+    inputs:
+      runs-on-value:
+        required: true
+        type: string
+    secrets:
+      DOCKER_HUB_LOGIN:
+        required: true
+      DOCKER_HUB_PASSWORD:
+        required: true
+
+jobs:
+  check:
+    name: Run checks
+    runs-on: ${{ inputs.runs-on-value }}
+    env:
+      HOME: /root
+    steps:
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: 1.19
+
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.11.2
+
+      - name: Build
+        shell: bash
+        run: |
+          make build
+
+      - name: Run tests
+        shell: bash
+        run: |
+          [ $(uname -m) = x86_64 ] && curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.19.0/kind-linux-amd64
+          chmod +x ./kind
+          sudo mv ./kind /usr/local/bin/kind
+          kind create cluster --retain -v 100
+          kubectl get pod
+          kubectl apply -f https://github.com/cert-manager/cert-manager/releases/download/v1.11.0/cert-manager.yaml
+
+          make helm-kind-install
+          docker pull ytsaurus/ytsaurus-nightly:dev-23.1-5f8638fc66f6e59c7a06708ed508804986a6579f
+          kind load docker-image ytsaurus/ytsaurus-nightly:dev-23.1-5f8638fc66f6e59c7a06708ed508804986a6579f
+          docker pull ytsaurus/ytsaurus-nightly:dev-23.1-9779e0140ff73f5a786bd5362313ef9a74fcd0de
+          kind load docker-image ytsaurus/ytsaurus-nightly:dev-23.1-9779e0140ff73f5a786bd5362313ef9a74fcd0de
+          docker pull ytsaurus/ytsaurus-nightly:dev-23.2-62a472c4efc2c8395d125a13ca0216720e06999d
+          kind load docker-image ytsaurus/ytsaurus-nightly:dev-23.2-62a472c4efc2c8395d125a13ca0216720e06999d
+          YTSAURUS_ENABLE_E2E_TESTS=true make test
+          helm uninstall ytsaurus
+
+          ./compat_test.sh --from-version 0.4.1 --to-version trunk
+
+      - name: Docker Hub login
+        run: |
+          echo '${{ secrets.DOCKER_HUB_PASSWORD }}' | docker login --username ${{ secrets.DOCKER_HUB_LOGIN}} --password-stdin
+
+      - name: Docker
+        shell: bash
+        run: |
+          make release RELEASE_VERSION=0.0.$(git rev-list --count HEAD)-dev-${GITHUB_SHA} OPERATOR_IMAGE=ytsaurus/k8s-operator-nightly OPERATOR_CHART=ytop-chart-nightly
```

Comment:
Release to docker hub should be done only on commits to `main` branch

Reply:
Fixed